baseplate/context/memcache/lib.py

```diff
@@ -0,0 +1,108 @@
+from six import string_types
```

Comment:
there's a `_compat` module in the root of baseplate for this stuff. six is not a dependency right now.